code/controllers/subsystem/ticker.dm

```diff
@@ -525,7 +528,10 @@ SUBSYSTEM_DEF(ticker)
 		if(STATION_DESTROYED_NUKE)
 			news_message = "We would like to reassure all employees that the reports of a Syndicate backed nuclear attack on [station_name()] are, in fact, a hoax. Have a secure day!"
 		if(STATION_EVACUATED)
-			news_message = "The crew of [station_name()] has been evacuated amid unconfirmed reports of enemy activity."
+			if(emergency_reason)
+				news_message = "[station_name()] has been evacuated after transmitting the following distress beacon:\n\n[emergency_reason]"
```

Comment:
Make sure you can't do XSS with this, like calling the shuttle with the reason `<h1>funny</h1>`

Reply:
On a look through of the code, it looks like it is html_encoded'd anyway, so nevermind.

Reply:
Not an issue, it's handled through minor_announce which uses html_encode

Tested locally to be sure, just spits out the text
![image](https://user-images.githubusercontent.com/39933245/94350173-4fe84c00-0011-11eb-90df-c6d170130005.png)